src/shared/components/schoolCard/schoolCard.css

```diff
@@ -2,10 +2,10 @@
   display: flex;
   border-radius: 4px;
   align-items: center;
-  margin: 20px 10px;
+  margin: 25px 10px;
```

Comment:
Wow we must have way different screens or something. I tested in Chrome and Firefox, used the mobile inspector and I just totally disagree. It may not be perfect, but it's a vast improvement on what's currently on the site. We should do a screenshare and clear this up.